dex-it/src/test/scala/com/wavesplatform/it/matcher/api/http/debug/GetOldestSnapshotOffsetSpec.scala

```diff
@@ -30,7 +30,7 @@ class GetOldestSnapshotOffsetSpec extends MatcherSuiteBase with ApiKeyHeaderChec
         mkOrder(alice, wavesUsdPair, BUY, 10.waves, 3.usd)
       ).foreach(placeAndAwaitAtDex(_))
 
-      validate200Json(dex1.rawApi.getOldestSnapshotOffset) should be(-1)
+      validate200Json(dex1.rawApi.getOldestSnapshotOffset) should be(0)
```

Comment:
Probably we need to wrap this to `eventually`, because a snapshot might not be created in this time